exercises/queen-attack/src/example/java/QueenAttackCalculator.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Arrays;
+
+public final class QueenAttackCalculator {
+
+    private final int[] queen1Coordinates;
+
+    private final int[] queen2Coordinates;
+
+    public QueenAttackCalculator(final int[] queen1Coordinates, final int[] queen2Coordinates)
```

Comment:
Just as an FYI: I'm proposing to the group to switch to SAN: exercism/x-common#446.  That is a significant change and should *not* hold up this work.